src/doc/unstable-book/src/field-init-shorthand.md

```diff
@@ -1,10 +0,0 @@
-# `field_init_shorthand`
-
-The tracking issue for this feature is: [#37340]
-
-[#37340]: https://github.com/rust-lang/rust/issues/37340
-
-------------------------
```

Comment:
so, I purposefully put these at the end of sections that do not have docs, just to keep it consistent; would you mind updating the ones you've added to have this?